src/SlackMessages.ts

```diff
@@ -82,8 +80,20 @@ export function atEveryone() {
     return "<!everyone>";
 }
 
+function hasItems(arr: any[]) {
+    return arr != null && arr.length > 0;
```

Comment:
I knew you had to check for `null` before calling `length`, but I did not know that `!= null` was shorthand for `!== null && !== undefined`. That's good to know, thanks.